QuotesApp-shalenMathew/app/src/main/java/com/example/quotesapp/data/QuoteRepositoryImplementation.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.quotesapp.data
+
+import com.example.quotesapp.domain.model.Quote
+import com.example.quotesapp.domain.repository.QuoteRepository
+
+class QuoteRepositoryImplementation:QuoteRepository {
+    override fun getQuote(): MutableList<Quote> {
```

Comment:
why returning mutable?

Reply:
the library i'm using for for swiping cards decks requires a mutable list

Reply:
if we think about scaling, what if in future library changes?

the place where we are using library, there we can keep it as mutable, WDYT?